src/portal/src/lib/components/tag/tag.component.html

```diff
@@ -159,7 +159,7 @@ <h3 class="modal-title">{{ 'REPOSITORY.RETAG' | translate }}</h3>
             </clr-dg-row>
             <clr-dg-footer>
                 <span *ngIf="pagination.totalItems">{{pagination.firstItem + 1}} - {{pagination.lastItem + 1}} {{'REPOSITORY.OF' | translate}}</span> {{pagination.totalItems}} {{'REPOSITORY.ITEMS' | translate}}&nbsp;&nbsp;&nbsp;&nbsp;
-                <clr-dg-pagination #pagination [clrDgPageSize]="10"></clr-dg-pagination>
+                <clr-dg-pagination #pagination [clrDgPageSize]="50"></clr-dg-pagination>
```

Comment:
<clr-dg-pagination #pagination [clrDgPageSize]="pageSize">
<clr-dg-page-size [clrPageSizeOptions]="[10,20,50,100]">

Reply:
Have discussed with pm, he agrees with 50 per page, what you are talking about is just another solution